torch/csrc/jit/stack_functions.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+#include "ATen/ATen.h"
+#include "torch/csrc/jit/tensor_conversions.h"
+
+namespace torch { namespace jit {
+
+using Stack = std::vector<at::Tensor>;
+using Operation = std::function<int(Stack&)>;
```

Comment:
fun fact: this is a different definition than that in interpreter.h (equivalent but different). can we deduplicate them?